### PR TITLE
Fix distances to have two decimal places

### LINKED ISSUE #632: Group location distance not correct
## Acceptance Criteria

- [ ] It should tell me the distance the group is from me.

## Context

Given that I'm looking for a group when I see the results then it should show me how far away that group is.

### PR DESCRIPTION
**Pull request by richarddubay**
Fixes #632 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [ ] Run `test` and make sure all tests pass
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

File: src/group-finder/GroupCard.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const GroupCard = ({
               <ScheduleText>{schedule.description}</ScheduleText>
             ) : null}
 
-            <DistanceText>{parseInt(distance, 0)} miles away</DistanceText>
+            <DistanceText>{parseFloat(distance).toFixed(2)} miles away</DistanceText>
```
[the remark this answers is not in the document]
Comment (conrad-vanl) [reply src/group-finder/GroupCard.js]: or use lodash's `round` https://lodash.com/docs#round

[the remark this answers is not in the document]
Comment (conrad-vanl) [reply src/group-finder/GroupCard.js]: image

[the remark this answers is not in the document]
Comment (conrad-vanl) [reply src/group-finder/GroupCard.js]: could do something like: `round(distance || 0)`

[the remark this answers is not in the document]
Comment (richarddubay) [reply src/group-finder/GroupCard.js]: @IsaacHardy I could make the change in a different PR to make this `round(distance || 0)` if you think that would be a better solution.

[the remark this answers is not in the document]
Comment (IsaacHardy) [reply src/group-finder/GroupCard.js]: @richarddubay I think that is a great idea.

**Pull request merged**